adapter/config/parser.go

```diff
@@ -42,7 +42,7 @@ var (
 )
 
 // DefaultGatewayName represents the name of the default gateway
-const DefaultGatewayName = "Default"
+const DefaultGatewayName = "default"
```

Comment:
does this have any usages now? shall we remove this from the code?

Reply:
no, removed and resolved